doc/spec/selection-button.md

```diff
@@ -0,0 +1,59 @@
+## Provides the suggested markup for the selection buttons
+
+* Specification for https://jira.ez.no/browse/EZP-27572
+
+ Goal here is to provide activable elements (like buttons) that are enabled and can triggers actions only if some elements of a list are selected.
+ The selection is done with the help of checkboxes or radio buttons related to items of the list.
+ Some buttons need at least one or more element checked to be enabled, but others will need one and only one element checked to be enabled.
+
+
+# Markup
+
+* Here is a simple markup example of a selection button that can be enabled with one or more element and that can be enabled by every checkbox of <ez-selection-activate-element> :
```

Comment:
instead of a list, add sub title, like:
```md
## Enable a button based on a checkboxes

...

## Enable a button only if one element is selected

...

## Enable a button based on a subset of checkboxes

...

```